<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameLoader</class>
 <widget class="QWidget" name="GameLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#GameLoader{
background-image: url(:/resources/img/backgrounds/gameLoaderBackground.jpg);
}

QScrollBar:horizontal
 {
     height: 15px;
     margin: 3px 15px 3px 15px;
     border: 1px transparent #2A2929;
     border-radius: 4px;
     background-color: yellow; 
 }

 QScrollBar::handle:horizontal
 {
     background-color: blue;
     min-width: 5px;
     border-radius: 4px;
 }

 QScrollBar::add-line:horizontal
 {
     margin: 0px 3px 0px 3px;
     width: 10px;
     height: 10px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:horizontal
 {
     margin: 0px 3px 0px 3px;
     height: 10px;
     width: 10px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }

 QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
 {
     height: 10px;
     width: 10px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }


 QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on
 {
     height: 10px;
     width: 10px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }

 QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
 {
     background: none;
 }


 QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
 {
     background: none;
 }

 QScrollBar:vertical
 {
     background-color: #000000;
     width: 15px;
     margin: 15px 3px 15px 3px;
     border: 1px black #2A2929;
     border-radius: 4px;
 }

 QScrollBar::handle:vertical
 {
     background-color: #cdcdcd;         /* #605F5F; */
     min-height: 5px;
     border-radius: 4px;
 }

 QScrollBar::sub-line:vertical
 {
     margin: 3px 0px 3px 0px;
     height: 10px;
     width: 10px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

 QScrollBar::add-line:vertical
 {
     margin: 3px 0px 3px 0px;
     height: 10px;
     width: 10px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
 {
     height: 10px;
     width: 10px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }


 QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
 {
     height: 10px;
     width: 10px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
 {
     background: none;
 }


 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
 {
     background: none;
 }

QTableWidget {background-color: rgba(117, 61, 0,190);
outline: 0;
color: #dfdfdf;
}
QHeaderView::section {background-color: transparent;}
QHeaderView {background-color: rgba(69, 36, 1,180);
color: #dfdfdf;
font-size: 18px;
}
QTableCornerButton::section {background-color: transparent;}
QTableWidget::item:selected{ 
background-color: rgba(214, 137, 56,170); 
}</string>
  </property>
  <widget class="QTableWidget" name="maps">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>651</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#maps{
border: 4px solid black;
border-radius: 15px;
color: #dfdfdf;
}

#maps::item{
border-bottom: 1px solid white;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>15</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Pályanév</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Méret</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nehézség</string>
    </property>
   </column>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>Első pálya</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>12x12</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>Könnyű</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Második pálya</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>15x15</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>Közepes</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>560</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="prevBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>560</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>560</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}

QPushButton:disabled[flat=true]{
background-color: rgba(74, 68, 68,170);
}</string>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="PaintedWidget" name="playerPortraitWidget" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>230</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#playerPortraitWidget{
background-color: rgba(0,0,0,120);
border: 2px solid black;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerNameLabel">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>190</y>
     <width>180</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#playerNameLabel{
color: #dfdfdf;
}</string>
   </property>
   <property name="text">
    <string>PlayerName</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>650</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintedWidget</class>
   <extends>QWidget</extends>
   <header>paintedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
